src/equations/compressible_euler_1d.jl

```diff
@@ -375,6 +375,17 @@ end
   λ_max = max(v_mag_ll, v_mag_rr) + max(c_ll, c_rr)
 end
 
+"""
+    flux_ranocha(u_ll, u_rr, orientation_or_normal_direction, equations::AbstractEquations{1})
+
+Enables calling `flux_ranocha` with non-integer arguments `orientation_or_normal_direction` for
+one-dimensional equations.
+
+Returns `orientation_or_normal_direction * flux_ranocha(u_ll, u_rr, 1, equations)`.
+"""
+@inline function flux_ranocha(u_ll, u_rr, orientation_or_normal_direction, equations::AbstractEquations{1})
```

Comment:
```suggestion
@inline function flux_ranocha(u_ll, u_rr, normal_direction::AbstractVector, equations::AbstractEquations{1})
```
This is more specific and should clarify the purpose. The docstring should be changed as well.
I'm not quite sure whether we should extend it for general equations like this. In the end, you use it only for the compressible Euler equations. Since we often try to collect related functionality explicitly, it would probably make sense to define this only for Euler and move it there. What do you think?

Reply:
That sounds good to me. I'll restrict to just compressible Euler for now.